CHANGELOG.md

```diff
@@ -1,38 +0,0 @@
-<!--
```

Comment:
Why have you removed changelog?

Reply:
Occasionally maybe. Got it back, added changes I have made.